wavelength-plugins/ssb.js

```diff
@@ -605,419 +609,435 @@ try {
-				case 'pending':
-					if (WL.ssb[target[1]].bought[target[2]] === true) return this.errorReply(target[1] + '\'s ' + target[2] + ' is already ' + target[3] + '.');
+				case "pending":
+					if (WL.ssb[target[1]].bought[target[2]] === true) return this.errorReply(`${target[1]}'s ${target[2]} is already ${target[3]}.`);
 					WL.ssb[target[1]].bought[target[2]] = true;
 					writeSSB();
-					return this.sendReply(target[1] + '\'s ' + target[2] + ' was marked as pending.');
+					return this.sendReply(`${target[1]}'s ${target[2]} was marked as pending.`);
 					//break;
-				case 'remove':
-					if (WL.ssb[target[1]].bought[target[2]] === false) return this.errorReply(target[1] + '\'s ' + target[2] + ' is already removed.');
-					if (!target[4] || toId(target[4]) !== 'force') return this.sendReply('WARNING. If you remove this purchase the user will not be able to use their ' + target[2] + ' and the user will not be refunded (unless you provide it). If you are sure you want to do this, run: /ssb log mark, ' + target[1] + ', ' + target[2] + ', ' + target[3] + ', force');
+				case "remove":
+					if (WL.ssb[target[1]].bought[target[2]] === false) return this.errorReply(`${target[1]}'s ${target[2]} is already removed.`);
+					if (!target[4] || toId(target[4]) !== "force") return this.sendReply(`WARNING. If you remove this purchase the user will not be able to use their ${target[2]} and the user will not be refunded (unless you provide it). If you are sure you want to do this, run: /ssb log mark, ${target[1]}, ${target[2]}, ${target[3]}, force`);
 					WL.ssb[target[1]].bought[target[2]] = false;
 					writeSSB();
-					return this.sendReply(target[1] + '\'s ' + target[2] + ' was removed.');
+					return this.sendReply(`${target[1]}'s ${target[2]} was removed.`);
 					//break;
 				default:
-					return this.parse('/help ssb log');
+					return this.parse("/help ssb log");
 				}
 				//break;
 			default:
-				return this.parse('/help ssb log');
+				return this.parse("/help ssb log");
 			}
 		},
-		loghelp: ['/ssb log - Accepts the following commands:',
-			'/ssb log view, [all|user] - View the purchases of a user or all users. Requires &, ~ unless viewing your own.',
-			'/ssb log mark, [user], [cItem|cAbility|cMove], [complete|pending|remove] - Update the status for a users SSBFFA purchase.',
+		loghelp: [
+			`/ssb log - Accepts the following commands:
+			/ssb log view, [all|user] - View the purchases of a user or all user's. Requires &, ~ unless viewing your own.
+			/ssb log mark, [user], [cItem|cAbility|cMove], [complete|pending|remove] - Update the status for a user's SSBFFA purchase.`,
 		],
-		forceupdate: 'validate',
-		validateall: 'validate',
+
+		forceupdate: "validate",
+		validateall: "validate",
 		validate: function (target, room, user, connection, cmd, message) {
-			if (!this.can('ssb')) return;
-			if (!target && toId(cmd) !== 'validateall') return this.parse('/help ssb validate');
+			if (!this.can("ssbffa")) return;
```

Comment:
Pretty sure the config file permission is ssb not ssbffa. need @HoeenCoder's confirmation

Reply:
oh it is my bad :x